studio/components/interfaces/BillingV2/Subscription/Subscription.tsx

```diff
@@ -23,6 +25,35 @@ const Subscription = ({}: SubscriptionProps) => {
   const panel = router.query.panel
   const isOrgBilling = !!organization?.subscription_id
 
+  useEffect(() => {
```

Comment:
should we do this on the project usage page as well?

Reply:
Let's do that in a separate PR